zilencer/lib/stripe.py

```diff
@@ -74,7 +74,7 @@ def wrapped(*args: Any, **kwargs: Any) -> Any:
             raise StripeError(
                 "Missing Stripe config. See https://zulip.readthedocs.io/en/latest/subsystems/billing.html.")
 
-        if not Plan.objects.exists():
+        if settings.DEVELOPMENT and not Plan.objects.exists():
```

Comment:
We probably don't want this in the TEST_SUITE case either, right? I'm a bit surprised that this doesn't get triggered in our current tests.